apps/total-typescript/package.json

```diff
@@ -105,7 +105,7 @@
     "react-use-clipboard": "^1.0.7",
     "refractor": "^4.5.0",
     "speakingurl": "^14.0.1",
-    "superjson": "^1.9.1",
+    "superjson": "^1.11.0",
```

Comment:
version mismatches cause all sorts of issues in monorepo 😭

Reply:
There's a package called manypkg that can help

Reply:
ya it's great! it's installed in this repo, just sometimes forget/neglect and get bitten (again) ☠️